site-content/source/modules/ROOT/pages/blog.adoc

```diff
@@ -14,7 +14,31 @@ NOTES FOR CONTENT CREATORS
 [openblock,card-header]
 ------
 [discrete]
-=== Arithmetic Operators in Cassandra 4.0
+=== Configurable Storage Ports and Why We Need Them
+[discrete]
+==== December 28, 2021
```

Comment:
Not sure when this can be published but just noting that the date might need to be updated. 👍